lib/config_upgrader.js

```diff
@@ -235,8 +236,9 @@ class ConfigUpgrader {
       const config = new Config({
         configFile: configFile,
         cwd: process.cwd()
-      });
-      upgrader.upgrade(config.useSpecificConfig);
+      },
+      new Linter());
```

Comment:
is this indentation correct? It took me a moment, reading it, to realize that this value is an argument to `new Config`